bigbluebutton-html5/imports/api/meetings/server/modifiers/addMeeting.js

```diff
@@ -5,6 +5,7 @@ import {
 } from 'meteor/check';
 import Meetings from '/imports/api/meetings';
 import Logger from '/imports/startup/server/logger';
+import createNote from '/imports/api/note/server/methods/createNote'
```

Comment:
I particularly didn't like this `createNote` inside `addMeeting`. Is there a better place for it? Could use some guidance here.